app/models/legacy_course.rb

```diff
@@ -0,0 +1,17 @@
+class LegacyCourse < Course
+  def wiki_edits_enabled?
+    false
+  end
+
+  def wiki_title
+    prefix = 'Education_Program:'
+    escaped_slug = slug.tr(' ', '_')
+    "#{prefix}#{escaped_slug}"
+  end
+
+  def update(data={}, should_save=true)
+    # For legacy courses, the update may involve pulling data from MediaWiki
+    require "#{Rails.root}/lib/course_update_manager"
```

Comment:
This convention of inlining the requires was introduced by @ntdb, and I've found — especially lately — that it's quite nice when it comes to refactoring. In this specific case, it doesn't make a big difference... but have that require be inline for the older version of this method (on course.rb) made it easy to see that this was the only place that needed that... and so there was no need to double and triple check the top level requires.